tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/WholeStageExecParser.scala

```diff
@@ -45,14 +45,18 @@ case class WholeStageExecParser(
     // if any of the execs in WholeStageCodegen supported mark this entire thing
     // as supported
     val anySupported = childNodes.exists(_.isSupported == true)
+    val unSupportedExecsArray = childNodes.filter(_.unsupportedExecs.nonEmpty).map(
+      x => x.exec)//.toArray
+    val unSupportedExecs = unSupportedExecsArray.mkString(";")
```

Comment:
we used ";" for csv?

Reply:
We use ":" and ";" for csv i.e readFormat uses ":" and  writeDataFormat uses ";" . I went ahead with ";" as it did not require updating the tests.